done/static/html/done.html

```diff
@@ -1,10 +1,12 @@
-<div class="done_block">
-  <div class="done_left_spacer"> </div>
-  <div class="done_windshield"> 
-    <div style="display:block; width:100%; height:6px"></div>
-    <div class="done_wipe done_wipe_off"> Undo </div>
-    <div class="done_wiper"> <span class="ui-icon ui-icon-circle-plus ui_icon_white" style="padding-bottom:2px"></span> </div>
-    <div class="done_wipe done_wipe_on"> Mark as complete </div>
+<div class="done_onoffswitch_wrapper">
+  <div class="done_left_spacer"></div>
+  <div class="done_onoffswitch">
+    <input type="checkbox" name="onoffswitch" class="done_onoffswitch-checkbox" id="{id}_myonoffswitch" checked="{done}">  </input>
+    <label class="done_onoffswitch-label" for="{id}_myonoffswitch">
+        <span class="done_aria_hidden">I have completed this</span>
```

Comment:
We cannot. XBlocks are intended to be independent of the edX platform. Until/unless sr is support in xblock workbench and part of the XBlocks interface, this would either break things when developing in the SDK, or risk us overriding edX CSS. 

One of the XBlock stories is about theming. It's not a lot of work, but it has to be done before we can have this kind of reuse.